apps/happy-blocks/block-library/universal-header/index.php

```diff
@@ -445,5 +447,20 @@
 				<?php endif; ?>
 				<form class="<?php echo ! $args['include_site_title'] ? 'happy-blocks_inner_search' : ''; ?>"  role="search" method="get" action=""><label for="wp-block-search__input-1" class="screen-reader-text"><?php echo esc_html( $args['search_placeholder'] ); ?></label><div class="happy-blocks-search__inside-wrapper"><input type="search" id="wp-block-search__input-1" name="s" value="" placeholder="<?php echo esc_html( $args['search_placeholder'] ); ?>"></div></form>
 			</div>
+		</div>
 	</div>
-</div>
+	<?php elseif ( 'learn' === $args['active_tab'] ) : ?>
+	<div class="happy-blocks-mini-search happy-blocks-header__<?php echo esc_html( $args['active_tab'] ); ?>">
```

Comment:
Given that the only difference between `support` and `learn` is the `happy-blocks-header__learn` class, what do you think of only adding it conditionally on learn instead of duplicating the code?

Reply:
100% agree 👍